reference/dataset-from-filter-params.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: "Dataset From Filter Params"
+api: "POST https://api.promptlayer.com/dataset-from-filter-params"
+---
+
+This endpoint allows you to create a dataset based on a specific search query. This dataset is then stored under a given workspace, associated with specified tags, metadata, and other criteria such as time range and scores.
```

Comment:
"This endpoint allows you to create a dataset based on a historical search query. Search filters can include tags, metadata, and other criteria such as time range and scores. This dataset is then stored in a given workspace."